docs/serving/samples/tag-header-based-routing/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Tag Header Based Routing"
+linkTitle: "Tag Header Based Routing"
```

Comment:
No need for linkTitle when this is the same as title - can be removed.

Reply:
Done